domains/measurements/Measurements.API/API/Models/Quality.cs

```diff
@@ -0,0 +1,15 @@
+using System.ComponentModel;
+
+namespace API.Models;
+
+public enum Quality
+{
+    [Description("Measured")]
+    Measured = 10,
+    [Description("Revised")]
+    Revised = 20,
+    [Description("Calculated")]
+    Calculated = 30,
+    [Description("Estimated")]
+    Estimated = 40,
+}
```

Comment:
```suggestion
    [JsonConverter(typeof(JsonStringEnumConverter))]
    public enum EnergyQuantityValueQuality
    {
        measured,
        revised,
        calculated,
        estimated,
    }
```

fix compile time errors because of renames